py3/bluesky-0.4.3/meta.yaml

```diff
@@ -40,9 +40,13 @@ test:
     - bluesky
     - bluesky.broker_callbacks
     - bluesky.broker_examples
+    - bluesky.callbacks
+    - bluesky.callbacks.core
+    - bluesky.callbacks.olog
```

Comment:
It's working because you only import pyolog inside of the callback
https://github.com/NSLS-II/bluesky/blob/master/bluesky/callbacks/olog.py#L148